rollup.config.js

```diff
@@ -40,13 +44,14 @@ const baseConfig = {
 const baseExternal = ['react', 'prop-types', 'preact']
 
 const mainConfig = Object.assign({}, baseConfig, {
-  external: baseExternal.concat([
-    'emotion',
-    'emotion-utils',
-    'hoist-non-react-statics',
-    'stylis-rule-sheet',
-    '@emotion/is-prop-valid'
-  ]),
+  external: baseExternal.concat(
+    uniq(
+      flatMap(
+        dir => Object.keys(require(`${dir}/package.json`).dependencies || {}),
```

Comment:
This should include `peerDependencies` too.

Reply:
good catch, fixed